lite/backends/nnadapter/nnadapter/driver/huawei_ascend_npu/utility.cc

```diff
@@ -33,6 +34,31 @@ void InitializeAscendCL() {
   static bool initialized = false;
   mtx.lock();
   if (!initialized) {
+    NNADAPTER_VLOG(5)
+        << "Get the installed version of the local Ascend CANN...";
+    std::tuple<int, int, int, int> ascend_cann_versions =
+        GetAscendCANNVersion();
+    int major_version, minor_version, patch_version, bugfix_version;
+    std::tie(major_version, minor_version, patch_version, bugfix_version) =
+        ascend_cann_versions;
+    NNADAPTER_VLOG(5) << "The installed version of the local Ascend CANN is "
```

Comment:
NNADAPTER_LOG(WARNING) << "CANN version mismatch. The build version is xxx but the current environment version is xxxx.".

Reply:
done